enhancements/api-review/add-new-CRD-ImageDigestMirrorSet-and-ImageTagMirrorSet-to-config.openshift.io.md

```diff
@@ -326,6 +326,30 @@ Phase 2:
 - components listed in the above [document](https://docs.google.com/document/d/11FJPpIYAQLj5EcYiJtbi_bNkAcJa2hCLV63WvoDsrcQ/edit?usp=sharing) migrate to new APIs ImageDigestMirrorSet, ImageTagMirrorSet
 - register and expose the CRDs.
 
+#### Implementation updates
+Actual implementation for migration path of Phase 2 was different from the original design:
+
+The webhook is not implementable. It cannot convert between different kind of resources. 
+
+If copy existing ImageContentSourcePolicy content to ImageDigestMirrorSet allow both kind of objecting existing, this meeting [notes](https://docs.google.com/document/d/13h6IJn8wlzXdiPMvCWlMEHOXXqEZ9_GYOl02Wldb3z8/edit?usp=sharing) raised the concern that in some cases the final `registries.conf` will lead the customer implying only one of the object is honored.
```

Comment:
Let's not link to google doc here and instead summarize the relevant portion inline.